src/lib/timers/ingress_drop_monitor.lua

```diff
@@ -26,8 +27,9 @@ function new(args)
       tips_url = args.tips_url or default_tips_url,
       last_flush = 0,
       last_value = ffi.new('uint64_t[1]'),
-      current_value = ffi.new('uint64_t[1]')
+      current_value = ffi.new('uint64_t[1]'),
    }
+   ret.threshold = 1
```

Comment:
This is from testing I guess; please remove.  Likewise remove the trailing comma on line 30